addons/docs/src/blocks/index.ts

```diff
@@ -12,9 +12,7 @@ export * from './ExternalDocsContainer';
 export * from './DocsStory';
 export * from './Heading';
 export * from './Meta';
-export * from './Preview';
```

Comment:
Can we roll this back & do it separately as part of the doc blocks work? It's going to cause conflicts

Reply:
sure thing!